README.md

```diff
@@ -37,6 +37,20 @@ val gson: Gson = GsonBuilder()
 
 If you already have other type adapter factories registered, be sure place `KotlinReflectiveTypeAdapterFactory` at the beginning of the list.
 
+When calling `create`, an optional `enableDefaultPrimitiveValues` parameter is available (default
+ is `false`). When this property is set to `true`, nonnull primitives will automatically receive
+ default values. The table below shows which primitives are supported.
+ 
+| Primitive Type | Default Value |
+| --- | --- |
+| Boolean | False |
+| Byte | 0 |
+| Double | 0 |
+| Float | 0 |
+| Int | 0 |
+| Long | 0 |
+| Short | 0 |
```

Comment:
`Char` is sad that it was left out

Reply:
Char actually defaults to null. Do you think that I should still mention it?

Reply:
I felt it weird since that will still fail.

Reply:
Ah, I meant that `char` is the 8th primitive type, so it should probably be handled as well.

Reply:
So I did a little extra testing. `Char` has a default value of `\u0000` which is null. But apparently that is still a valid value for a `char`.

I went and added support for it.